packages/perseus/src/widgets/interactive-graphs/reducer/initialize-graph-state.ts

```diff
@@ -344,3 +352,48 @@ function getCircleCoords(graph: PerseusGraphTypeCircle): {
         radiusPoint: [2, 0],
     };
 }
+
+const getAngleCoords = (params: {
+    graph: PerseusGraphTypeAngle;
+    range: [x: Interval, y: Interval];
+    step: [x: number, y: number];
+}): [Coord, Coord, Coord] => {
+    const {graph, range, step} = params;
+    if (graph.coords) {
+        return graph.coords;
+    }
+
+    const {snapDegrees, angleOffsetDeg} = graph;
+    const snap = snapDegrees || 1;
+    let angle = snap;
+    while (angle < 20) {
+        angle += snap;
+    }
+    angle = (angle * Math.PI) / 180;
+    const offset = ((angleOffsetDeg || 0) * Math.PI) / 180;
+
+    let defaultCoords: [Coord, Coord] = [
+        [0.85, 0.5],
+        [0.5, 0.5],
+    ];
+
+    defaultCoords = normalizePoints(range, step, defaultCoords, true);
+
+    // @ts-expect-error - TS2345 - Argument of type 'number[]' is not assignable to parameter of type 'readonly Coord[]'. | TS2556 - A spread argument must either have a tuple type or be passed to a rest parameter.
```

Comment:
I wasn't sure about whether I should bother with this expect error that got carried over from the old code. It would require creating new properly typed functions for both magnitude and vector, and seemed like a rabbit hole that wasn't worth diving into yet.